src/app/components/candidate-request/candidate-request.component.html

```diff
@@ -90,40 +73,28 @@ <h2>{{ 'request.header' | translate }}</h2>
     <div class="job">
       <mat-form-field appearance="outline">
         <mat-label>{{ 'request.skills' | translate }}</mat-label>
-        <mat-select formControlName="skills">
-          <mat-option value="Frontend">Frontend</mat-option>
-          <mat-option value="Backend">Backend</mat-option>
-          <mat-option value="BA">BA</mat-option>
-          <mat-option value="Ui/Ux">Ui/Ux</mat-option>
+        <mat-select formControlName="primaryTechnologyId">
+          <mat-option *ngFor="let skill of sandboxChosen.stackTechnologies" [value]="skill !== null ? skill.id : ''">{{skill
+            !== null ? skill.name : ''}}</mat-option>
         </mat-select>
       </mat-form-field>
 
       <mat-form-field appearance="outline">
         <mat-label>{{ 'request.job' | translate }}</mat-label>
-        <input formControlName="job" matInput placeholder="Placeholder" />
+        <input formControlName="currentJob" matInput placeholder="Placeholder" />
       </mat-form-field>
     </div>
   </div>
 
   <div class="skills">
     <mat-form-field appearance="outline">
       <mat-label>{{ 'request.certificates' | translate }}</mat-label>
-      <textarea
-        formControlName="certificates"
-        [cdkTextareaAutosize]
-        class="skills_textarea"
-        matInput
-      ></textarea>
+      <textarea formControlName="professionaCertificates" [cdkTextareaAutosize] class="skills_textarea" matInput></textarea>
```

Comment:
can you fix class name here and in the styles to use dash instead of underline please